spec/boxr/webhook_validator_spec.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+def generate_signature(payload, timestamp, key)
+  message_as_bytes = (payload.bytes + timestamp.bytes).pack('U')
+  digest = OpenSSL::HMAC.hexdigest('SHA256', key, message_as_bytes)
+  Base64.encode64(digest)
+end
+
+describe Boxr::WebhookValidator do
```

Comment:
Then, here, to use the hook to skip tests, simple add `:skip_rest` !

```suggestion
describe Boxr::WebhookValidator, :skip_reset do